_inc/lib/class.core-rest-api-endpoints.php

```diff
@@ -81,6 +81,13 @@ public static function register_endpoints() {
 			'callback'            => __CLASS__ . '::get_plans',
 			'permission_callback' => __CLASS__ . '::connect_url_permission_callback',
 
+        ) );
```

Comment:
You seem to have a bit of a mix of tabs and spaces here and below. I'd recommend switching to tabs only for indenting (and spaces for alignment in arrays for example, like you did below).